src/Networks/Xds/Blockcore.Networks.Xds/XdsMain.cs

```diff
@@ -43,10 +43,10 @@ public XdsMain()
             this.MaxTimeOffsetSeconds = 25 * 60;
             this.DefaultBanTimeSeconds = 8000;
             this.MaxTipAge = 2 * 60 * 60;
-            this.MinTxFee = Money.Coins(0.00001m).Satoshi;
+            this.MinTxFee = Money.Coins(0.01m).Satoshi;
             this.MaxTxFee = Money.Coins(1).Satoshi;
```

Comment:
hard fork

Reply:
Why? This is set to be the same as absolutely min fee.
Besides blocks won't get rejected based on fee it's just mempool relay problems at worth case will get effected